codespell_lib/data/dictionary_rare.txt

```diff
@@ -6,6 +6,7 @@ automatized->automated
 automatizes->automates
 backword->backward
 backwords->backwards
+bale->able
```

Comment:
Maybe pale?

Reply:
And kale, dale, male etc. None of those letters are close to B though. I really just wanted to cover the transposing of two characters from one perfectly valid word ( https://en.wikipedia.org/wiki/Baler ) to another more likely one. I suspect male might be more likely than pale anyway if we went down that route.

Reply:
Pale just came to my mind, because there is a german dialect, which doesn't distinguish between b or p, and d or t :-D
Close on the keyboard would be vale. I just  wanted to prevent people from auto-correcting something they didn't want to (yes i know interactive mode).

Reply:
Could also suggest bale->bald ?

Reply:
Personally I don't think vaguely infrequent rare valid words should suggest similar but unrelated ones unless there are very strong arguments for it, bald seems the only likely typo we've come up with. I'd suggest vale is as infrequent as bale.

I think also the frequency stats are very telling (I wasn't too far off with vale; bald is still pretty rare):
http://app.aspell.net/lookup?dict=en_US&words=bale%0D%0Aable%0D%0Avale%0D%0Abald%0D%0Apale%0D%0Amale%0D%0Adale%0D%0Akale%0D%0Asale

Between those and the fact it's just a transposition, so not even a real typo, I'd really like to leave it as is.

I think the auto-correct solution is to add what we dropped when we went to multi dictionary and have everything in rare (and names and informal) offer corrections to itself automatically). I can pull it out to a separate PR until that's been done if that would keep you two happy.

Reply:
You obviously understand this project much better than I do; I don't have strong opinions either way.